test/config-require/fixtures/with-arguments/test.js

```diff
@@ -0,0 +1,15 @@
+import test from 'ava';
+
+import cjs from './required.cjs';
+import {receivedArgs} from './required.mjs';
+
+test('receives arguments from config', t => {
+	t.deepEqual(receivedArgs, ['hello', 'world']);
+	t.deepEqual(cjs.receivedArgs, ['goodbye']);
+});
+
+test('ok to load for side-effects', async t => {
```

Comment:
It took me some time to understand that this test case take adventage of requesting the `side-effect` file before (during test loads config).

Maybe test case could be renamed? Sth like `side-effects are execute at first with test loading` or sth similar?

Reply:
Yea I'm fine with renaming them, I didn't think too hard about it just wanted to get the tests done in the little time I had.